appium/webdriver/appium_connection.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# pylint: disable=too-many-lines,too-many-public-methods,too-many-statements,no-self-use
+
+import base64
+import platform
+
+from selenium.webdriver.remote.remote_connection import RemoteConnection
+from selenium import __version__
+
+from appium.common.helper import library_version
+
+
+class AppiumConnection(RemoteConnection):
+
+    @classmethod
+    def get_remote_connection_headers(cls, parsed_url, keep_alive=False):
```

Comment:
should we override the method completely or call `super` and only modify the necessary fields?

Reply:
I would have liked to call `super` and override only `User-Agent`. But below cases raised errors..

So, I overrided getting the format from selenium webdriver.

```
super(AppiumConnection, AppiumConnection).__init__(cls, parsed_url, keep_alive) # init error in AppiumConnection
super(AppiumConnection, self).__init__(cls, parsed_url, keep_alive) # Error since this is classmethod
```

Reply:
I came up with the way to call super of classmethod what I would have liked to do in 85a9a1b

Reply:
I think you could simplify the call: `RemoteConnection. get_remote_connection_headers(cls, parsed_url, keep_alive=keep_alive)`
super call is useful when `self` instance is available, which is not the case for class methods

Reply:
the `override` notation is still needed in the docstring